manifests/kustomize/cluster/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
can we name the folder cluster-scoped-resource instead of cluster?

Reply:
Good idea.
Renaming just the top level one sounds good